### ISSUE TITLE
Allow upload in multiple folders

### ISSUE THREAD

**Issue by yaobinshi** [fossology/fossology#262]
---

Author Name: **Bob Gobeille** 
Original Redmine Issue: 6603, http://www.fossology.org/issues/6603
Original Date: 2014/02/27
Original Assignee: larry shi

---

The database structure allows one upload to be in multiple folders, but the GUI does not support this.  We should change the GUI.  This should simplify upload management since users can only browse what is under their root folder.

**Comment by mcjaeger** [comment]
yes, it would be good also to allow for two views, like one workflow view and one software component view (maybe sorted by alphabetical order or sorted by vendors)

**Comment by mcjaeger** [comment]
actually it should be possible with "copy of uploads"

**Comment by NesrinCeylan** [comment]
It would be nice if the user could assign the upload to a group directly in the upload process.
